vm_backtrace.c

```diff
@@ -119,16 +119,9 @@ typedef struct rb_backtrace_location_struct {
 	LOCATION_TYPE_CFUNC,
     } type;
 
-    union {
-	struct {
-	    const rb_iseq_t *iseq;
-            const VALUE *pc;
-	} iseq;
-	struct {
-	    ID mid;
-	    struct rb_backtrace_location_struct *prev_loc;
-	} cfunc;
-    } body;
+    const rb_iseq_t *iseq;
+    const VALUE *pc;
```

Comment:
Would it make sense to add a little bit of documentation here to explain what these things relate to?

Reply:
I think it's a good idea.  `iseq` is the instruction sequence for the method/block/etc. for the control frame. I think `pc` is the instruction pointer, but I don't actually know what it stands for or how it is used, only that we skip iseq frames without `pc` being set when generating the backtrace. I'm guessing @ko1 could provide a better explanation.